QuickTests/Helpers/QCKSpecRunner.h

```diff
@@ -10,7 +11,7 @@
  @param specClass The class of the spec to be run.
  @return An XCTestRun instance that contains information such as the number of failures, etc.
  */
-extern XCTestRun *qck_runSpec(Class specClass);
+QUICK_EXPORT XCTestRun *qck_runSpec(Class specClass);
```

Comment:
This is a nice touch, but the original problem was with using the Quick DSL in Objective-C++ files. This test helper isn't a part of the Quick DSL, so it might be outside of the scope of this pull request.

If we wanted to make sure this test helper could be used from within Quick's unit tests, even if those were written in Objective-C++ (they're all Swift or Objective-C at the moment), then I would argue that we should define a `QUICK_TESTS_EXPORT` macro. I don't think this file should have a dependency on `Quick/QCKDSL.h`.